WooCommerce/Classes/ViewRelated/Dashboard/Stats v4/StoreStatsPeriodViewModel.swift

```diff
@@ -237,6 +239,8 @@ private extension StoreStatsPeriodViewModel {
             return .redacted
         case .redacted:
             return .redacted
+        case .redactedDueToCustomRange:
+            return .redactedDueToCustomRange
```

Comment:
We might want to follow the JCP case to display the redacted view for the conversion stat and display the info icon only for the visitor stat.

Reply:
Added in c31913d 👍🏼